apps/api/src/queries/buildViewModelForResource/viewModels/index.ts

```diff
@@ -1,8 +1,8 @@
-export * from './event-sourced-vocabulary-list.view-model';
 export * from './playlist.view-model';
 export * from './song.view-model';
 export * from './tag.view-model';
 export * from './term.view-model.state-based';
 export * from './types/ViewModelId';
+export * from './vocabulary-list.view-model';
 // TODO do we want to keep barrel exporting given that we want to organize by feature going forward?
```

Comment:
We do need a big sweeping reorganization in the near future. Maybe when we rewrite each query service to be fully event sourced we should do this for the corresponding resource type?